api.md

```diff
@@ -469,7 +531,77 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
+
+Detach a data node from one hypertable or from all hypertables.
 
+This function is typically used if a data node should no longer be
+used by a hypertable. Then it need to be removed from all hypertables
+that use it.
+
+Another reason for detaching a data node from a hypertable is that
```

Comment:
The two reasons for detaching should be in list format

Reply:
Done